### ISSUE TITLE
In "RET lookup for the word", RET should be C-c C-o

### ISSUE THREAD

**Issue by xuchunyang** [agzam/mw-thesaurus.el#9]
https://github.com/agzam/mw-thesaurus.el/blob/aaa4b7b71b5b08535c294cad1517863167f97fa8/mw-thesaurus.el#L64

`org-open-at-point` is bound to `C-c C-o` not `RET` by default. Maybe you just want to bind `RET` directly without remap?

**Comment by agzam** [comment]
In evilified buffers RET is bound to `org-open-at-point`. So doing this I think will break it for Spacemacs and Doom users.

**Comment by xuchunyang** [comment]
I've never used Evil. If you want to `RET`, you can just use it, it should work for everybody, right? e.g.,

```elisp
(define-key mw-thesaurus-mode-map (kbd "RET") #'mw-thesaurus--lookup-at-point)
```

**Comment by agzam** [comment]
No, I'm pretty sure that's the reason why I did it that way in the first place. I wanted it to be in Org-mode, using outline worked, but it wasn't looking as nice, and in evilified org-mode buffer, `RET` wouldn't work as I wanted. I know, that's probably not the right way, but it works.

**Comment by xuchunyang** [comment]
You want `RET` to run `mw-thesaurus--lookup-at-point`, right?

and then

```elisp
(define-key mw-thesaurus-mode-map (kbd "RET") #'mw-thesaurus--lookup-at-point)
```

doesn't work for you?

**Comment by martianhi** [comment]
i also have this issue. not using evil, RET doesn't work for me.
